notes.txt

```diff
@@ -0,0 +1,241 @@
+# Record keeping
+
+Unfortunately I think this complicates unbonding logic a bit.
+Given this reality,
+1) I think the naive approach is to track user's unbondings in stakeibc module state, then transfer 
+the unbonded tokens from the delegation account to the user's account as soon as the unbonding period 
+ends. Staking module's unbonding-delegations object has completion_time: "2022-06-12T23:10:51.070726210Z" 
```

Comment:
Note: I am not sure, but seems like unbondings are triggered based on clock time rather than block height. This means we need to be able to check clock time reliably in epochs

Reply:
Will look into this - thanks for flagging